src/test/java/guitests/ContextMenuTests.java

```diff
@@ -81,6 +80,16 @@ public void test2() {
     }
 
     /**
+     * Tests selecting "Mark all below as read" and "Mark all below as unread" context menu items
+     */
+    @Test
+    public void markAllBelowAsReadUnread_tenIssuesInListView_markIssue9andBelowReadUnread() {
+        bringSelectedAndIssuesBelowInFocus(7);
+        markIssuesBelow(7, true);
+        markIssuesBelow(7, false);
```

Comment:
As mentioned in my previous review, this test can be stronger. Here you are testing two typical cases only. How do you know if calling the `mark as read` twice will not toggle the state back to unread? The test cases don't cover that.